stoiip_prob/main.py

```diff
@@ -21,3 +23,62 @@
 boi = np.array([1.01, 1.1, 1.12])
 
 print(np.around(stoiip(area, h, poro, swc, boi), 2))
+
+# %% Create random values for each distribution
```

Comment:
You need to add calculations with the uniform distribution

Reply:
Done